sql/moz-fx-data-shared-prod/udf/product_info_with_baseline/udf.sql

```diff
@@ -0,0 +1,95 @@
+CREATE OR REPLACE FUNCTION udf.product_info_with_baseline(legacy_app_name STRING, normalized_os STRING)
+RETURNS STRUCT<
+  app_name STRING,
+  product STRING,
+  canonical_app_name STRING,
+  canonical_name STRING,
+  contributes_to_2019_kpi BOOLEAN,
+  contributes_to_2020_kpi BOOLEAN,
+  contributes_to_2021_kpi BOOLEAN
+> AS (
+  CASE
+  WHEN
+    legacy_app_name LIKE "Focus iOS Baseline"
+    AND normalized_os LIKE "iOS"
+  THEN
+    STRUCT(
+      "focus_ios" AS app_name,
```

Comment:
This could also be `focus_ios_baseline`, but since this UDF is only used for smoot, which ignores `app_name`, it doesn't really matter. All that matter are `product` and `canonical_name`